packages/gatsby-source-drupal/src/utils.js

```diff
@@ -9,16 +9,17 @@ const {
 
 const { getOptions } = require(`./plugin-options`)
 
-const backRefsNamesLookup = new WeakMap()
-const referencedNodesLookup = new WeakMap()
+const backRefsNamesLookup = new Map()
```

Comment:
move to regular maps w/ IDs as we're not holding consistent references to the same node object anymore.